registry/handlers/api_test.go

```diff
@@ -31,8 +31,8 @@ import (
 	v2 "github.com/distribution/distribution/v3/registry/api/v2"
 	storagedriver "github.com/distribution/distribution/v3/registry/storage/driver"
 	"github.com/distribution/distribution/v3/registry/storage/driver/factory"
-	_ "github.com/distribution/distribution/v3/registry/storage/driver/testdriver"
 	"github.com/distribution/distribution/v3/testutil"
+	_ "github.com/docker/distribution/registry/storage/driver/inmemory"
```

Comment:
This should use the new location; using the old name causes the code from this repository (but the v2 release) to be vendored;


```go
	_ "github.com/distribution/distribution/v3/registry/storage/driver/inmemory"
```